QUARKUS-728.md

```diff
@@ -0,0 +1,68 @@
+# Quarkus-728 - Add support for OpenJDK 17
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-728
+
+The goal is to verify OpenJDK 17 integration
+
+## Scope of the testing
+Test development will focus on  
+ - New specifict scenarios will be created to cover the integration of new objects as “sealed” or “records” classes with several extensions as Hibernate or RESTEasy.
+ - Current scenarios must work with Java 17
+ - S2i (source to image) must works with Java 17  
+
+### Impact on testsuites and testing automation:
+ - Ensure this coverage works in JVM mode
```

Comment:
Please add subsection or new bullet point for enhancing GitHub Actions to cover both Java 11 and 17 in our "quarkus-qe" testsuites